mdsuite/calculators/einstein_helfand_ionic_conductivity.py

```diff
@@ -182,10 +182,10 @@ def ensemble_operation(self, ensemble):
         MSD of the tensor_values.
         """
         msd = tf.math.squared_difference(
-            tf.gather(ensemble, self.args.tau_values, axis=0), ensemble[None, 0]
+            tf.gather(ensemble, self.args.tau_values, axis=1), ensemble[:, 0, :]
         )
-        msd = self.prefactor * tf.reduce_sum(msd, axis=1)
-        self.msd_array += np.array(msd)  # Update the averaged function
+        msd = self.prefactor * tf.reduce_sum(msd, axis=2)
+        self.msd_array += np.array(msd)[0, :]
```

Comment:
Is there any advantage of adding `arr[0, :]` over `arr[0]` ? I guess readability?